cmd/process-agent/collector.go

```diff
@@ -67,7 +69,8 @@ func NewCollector(cfg *config.AgentConfig) (Collector, error) {
 		rtIntervalCh:  make(chan time.Duration),
 		cfg:           cfg,
 		groupID:       rand.Int31(),
-		apiClient:     api.NewClient(http.Client{Timeout: HTTPTimeout, Transport: cfg.Transport}, ReqCtxTimeout),
+		forwarder:     forwarder.NewDefaultForwarder(keysPerDomains(cfg.APIEndpoints)),
```

Comment:
This is a convenient way of setting things up :)

Reply:
Does the `DefaultForwarder` setup it's own `Transport` using the config? We'll want to make sure it does so that proxy settings continue to work

Reply:
Yes.  Per my comment above, it uses the proxy settings from the main agent config though.

My assumption was that if proxying was configured on the process agent then the same configuration would be applied to the main agent.

Reply:
It is - this is great :)